.dockstore.yml

```diff
@@ -413,3 +413,21 @@ workflows:
              - master
          tags:
              - /.*/
+   - name: GvsExtractCallsetPgen
+     subclass: WDL
+     primaryDescriptorPath: /scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl
+     filters:
+       branches:
+         - kl_pgen_extract_for_variants_pr
```

Comment:
are we going to need ah_var_store branches?

Reply:
First we'll need `EchoCallset` for `GvsExtractCallsetPgenMerged`, but we can add that when we merge this from `ah_var_store` to `EchoCallset` (separate from this PR).

Reply:
Ooo good catch!  Thank you for pointing that out.  I completely forgot to update that